### PR TITLE
Add SE-0315 to the changelog

### PR DESCRIPTION
**Pull request by Jumhyn**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,18 @@ _**Note:** This is in reverse chronological order, so newer entries are added to
 
 Swift 5.6
 ---------
+* [SE-0315][]:
+
+  Type expressions and annotations can now include "type placeholders" which
+  directs the compiler to fill in that portion of the type according to the usual
+  type inference rules. Typle placeholders are spelled as an underscore ("`_`") in
```
Comment (xedin) [inline CHANGELOG.md:12]: Typo: Typle

Comment (Jumhyn) [reply CHANGELOG.md:12]:
```suggestion
  type inference rules. Type placeholders are spelled as an underscore ("`_`") in
```

**Comment by xedin** [comment]
@slavapestov please take a look.

**Comment by xedin** [comment]
@swift-ci please smoke test

**Comment by Jumhyn** [comment]
@swift-ci please test

**Review by slavapestov — APPROVED** [review]

**Comment by Jumhyn** [comment]
@swift-ci please smoke test and merge

**Comment by Jumhyn** [comment]
@swift-ci please smoke test

**Comment by Jumhyn** [comment]
@swift-ci please smoke test macOS

**Comment by Jumhyn** [comment]
@swift-ci please smoke test macOS

**Comment by benrimmington** [comment]
@swift-ci Please smoke test macOS platform

**Comment by Jumhyn** [comment]
Thank you for getting this merged @benrimmington!

**Pull request merged**